tests/test6.c

```diff
@@ -1,15 +1,27 @@
+
+#define uthash_ptrdiff_t long
+
+#define uthash_noheaders
 #include "uthash.h"
+#include <string.h>
 #include <stdlib.h>   /* malloc */
 #include <stdio.h>    /* printf */
 
 /* Set up macros for alternative malloc/free functions  */
 #undef uthash_malloc
 #undef uthash_free
+#undef uthash_strlen
+#undef uthash_memcmp
 #define uthash_malloc(sz) alt_malloc(sz)
 #define uthash_free(ptr,sz) alt_free(ptr)
+#define uthash_strlen(s) alt_strlen(s)
+#define uthash_memcmp(v1,v2,s) alt_memcmp(v1,v2,s)
+
+static void * search_key = 0;
+static int seen_strlen = 0;
 
 typedef struct example_user_t {
-    int id;
+    char key[3];
```

Comment:
The platform I am specifically working with is Micrium on top of some Motorola CPU. There will substantial amount of work to get the tests in there, as they have to be converted into a single executable that is then rolled into a flash image. Since we haven't figured out how to even get debug output from our dev board, that will need to be converted to network calls. So, it's possible, but is a lot of work :)